crates/autopilot/src/shadow.rs

```diff
@@ -60,6 +68,7 @@ impl RunLoop {
             liveness,
             synchronization,
             current_block,
+            max_winners_per_auction,
```

Comment:
I'm a bit worried about some configuration error turning this feature on by accident. Could the constructor please panic if `max_winners_per_auction != 0` until we merge the last required DB change?

Reply:
Do you have a specific misconfiguration that could happen on your mind or is it just a gut feeling?
Because I think this is pretty safe and I think it's even riskier that I will forget to remove the assert you want to introduce.

Reply:
added assert